todo_app/app.py

```diff
@@ -1,11 +1,37 @@
-from flask import Flask
+from flask import Flask , render_template, request
+from flask.helpers import url_for
+from werkzeug.utils import redirect
+from todo_app.data.classes import to_do_item
+from todo_app.data.trello_items import  show_cards, add_card, doing_card
 
 from todo_app.flask_config import Config
 
+
+
 app = Flask(__name__)
 app.config.from_object(Config())
 
+@app.route('/' )
+def home():
+  trello_list = show_cards()
+  to_do = []
+  for list  in trello_list:
+    for card in list ['cards']:
+      id=card ['id']
+      title=card['name']
+      status=list['name']
+      item=to_do_item(id, title, status)
+      to_do.append(item)
+  return render_template('index.html', to_do=to_do)
+
+@app.route('/newitem', methods= ["POST"])
+def new_item():
+    item = request.form["title"]
+    add_card(item)
+    return redirect(url_for("home"))
 
-@app.route('/')
-def index():
-    return 'Hello World!'
+@app.route('/doingitem', methods= ["POST"])
```

Comment:
Blocker - I think this isn't actually being used in the frontend, or at least I couldn't see a way to mark a todo item as done